app/Providers/AppServiceProvider.php

```diff
@@ -21,6 +22,8 @@ class AppServiceProvider extends ServiceProvider
     public function boot()
     {
         Date::use(CarbonImmutable::class);
+        Schema::defaultStringLength(191);
```

Comment:
Why is this required?

Reply:
i have added because my maria db version

For example

`$table->string('first_name', 25);`

in this migration if i removed `25` i wont takes the default string length but i will revert that too.nope